### PR TITLE
Update menu ref issue 61

### PR DESCRIPTION
**Pull request by spdegabrielle**
Update menu as per  #61

### CODE REVIEW

**Comment by spdegabrielle** [comment]
as per #61

**Comment by spdegabrielle** [comment]
@Metaxal updated quickscript.scrbl

File: scribblings/quickscript.scrbl [hunk 1]
```diff
@@ -59,14 +59,14 @@ can add your own scripts there too if you think they may be useful to others.
...
-Click on the @gui{Scripts|Manage scripts|New script...} menu item, and enter @gui{Reverse} for the script name.
+Click on the @gui{Scripts|Manage|New script...} menu item, and enter @gui{Reverse} for the script name.
 This creates and opens the file reverse.rkt in the user's scripts directory.
 Also, a new item automatically appears in the @gui{Scripts} menu.
 
 In the .rkt file that just opened in DrRacket, modify the @racket[define-script] definition to the following:
 @margin-note{Don't name your script function @racket[reverse], it would shadow Racket's own and make the script hang.}
 @margin-note{If you later change the @racket[#:label] property, you will need to reload the menu by clicking on
-@gui{Scripts|Manage scripts|Reload scripts menu} after saving the file).}
+@gui{Scripts|Manage|Reload scripts menu} after saving the file).}
```
Comment (Metaxal) [inline scribblings/quickscript.scrbl:69]: This should be `Scripts|Manage|Reload menu`

File: scribblings/quickscript.scrbl [hunk 2]
```diff
@@ -266,7 +266,7 @@ There are some additional properties:
   If @racket[#f], the return value is not used.
 
   If this value is changed, make sure to reload the menu with
-  @gui{Scripts | Manage scripts | Reload menu}.
+  @gui{Scripts | Manage | Reload menu}.
```
Comment (Metaxal) [inline scribblings/quickscript.scrbl:269]: Remove spaces

File: scribblings/quickscript.scrbl [hunk 3]
```diff
@@ -295,12 +295,12 @@ There are some additional properties:
   If the script is persistent, the counter increases at each invocation of the script via the menu,
   whereas it always displays 1 if the script is not persistent.
 
-  @bold{Note:} Persistent scripts can be "unloaded" by clicking on the
-  @gui{Scripts|Manage scripts|Unload persistent scripts} menu item.
-  In the previous example, this will reset the counter. Make sure to unload a persistent script
+  @bold{Note:} Persistent scripts can be stopped by clicking on the
```
Comment (Metaxal) [inline scribblings/quickscript.scrbl:298]: It's not clear enough that's stopping a persistent script also resets it.
How about "can be stopped and reset"?

File: scribblings/quickscript.scrbl [hunk 4]
```diff
@@ -59,14 +59,14 @@ can add your own scripts there too if you think they may be useful to others.
 
 @section{Make your own script: First simple example}
 
-Click on the @gui{Scripts|Manage scripts|New script...} menu item, and enter @gui{Reverse} for the script name.
+Click on the @gui{Scripts|Manage|New script...} menu item, and enter @gui{Reverse} for the script name.
```
[the remark this answers is not in the document]
Comment (Metaxal) [reply scribblings/quickscript.scrbl:62]: and in other locations in the file

**Review by Metaxal — COMMENTED** [review]
Thanks for the PR! 

A few minor comments below.

File: scribblings/quickscript.scrbl (hunk 4 above)
[the remark this answers is not in the document]
Comment (spdegabrielle) [reply scribblings/quickscript.scrbl:62]: done

File: scribblings/quickscript.scrbl (hunk 1 above)
Comment (spdegabrielle) [reply scribblings/quickscript.scrbl:69]: done

File: scribblings/quickscript.scrbl (hunk 2 above)
Comment (spdegabrielle) [reply scribblings/quickscript.scrbl:269]: done

File: scribblings/quickscript.scrbl (hunk 3 above)
Comment (spdegabrielle) [reply scribblings/quickscript.scrbl]: done

File: tool.rkt [hunk 5]
```diff
@@ -335,14 +335,14 @@ It should then be very fast to load.
                   ("&Reload menu"                . ,(λ ()
                                                       (unload-persistent-scripts)
                                                       (reload-scripts-menu)))
-                  ("&Compile scripts and reload" . ,(λ ()
+                  ("&Compile scripts" . ,(λ ()
```
[the remark this answers is not in the document]
Comment (spdegabrielle) [reply tool.rkt:338]: done

**Comment by spdegabrielle** [comment]
@Metaxal updated quickscript.scrbl to 

- [ ] change the ... (3 separate dot characters) with … (a single character with 3 dots)?  (62)
- [ ] changed to ` Scripts|Manage|Reload menu` (69)
- [ ] Remove spaces (269)
- [ ] "can be stopped and reset" (298)

and tool.rkt to 

- [ ] Restore the alignment of the . ,(λ ()  (338)

bw

Stephen

**Comment by Metaxal** [comment]
Thanks Stephen!

**Pull request merged**